Add var.enabled to toggle on/off

c19 opened the pull request.

c19 committed:
Add enabled to toggle on/off

antonbabenko commented on outputs.tf:
```diff
@@ -3,9 +3,9 @@ output "ssh_user" {
 
 output "security_group_id" {
-  value = "${aws_security_group.bastion.id}"
+  value = "${aws_security_group.bastion.*.id}"
```
Thesу 2 outputs will return a list which is not backward-compatible. Please, wrap this with `element(concat(........), 0)`.

c19 committed:
backward-compatible

https://github.com/terraform-community-modules/tf_aws_bastion_s3_keys/pull/54#discussion_r259038503
